config/express.js

```diff
@@ -47,7 +47,7 @@ if (config.env === 'development') {
 }
 
 // mount all routes on /api path
-app.use('/api', routes);
+app.use('/', routes);
```

Comment:
o yes .There is this huge debate of keeping version numbers in URI or headers but I think having it at resource level makes sense.